parser/Parser/Parser.cpp

```diff
@@ -2,10 +2,29 @@
 //
 
 #include <iostream>
+#include <fstream>
+#include "CAFF_HEADER.h"
+#include "CAFF_CREDITS.h"
 
 int main()
 {
-    std::cout << "Hello World!\n";
+    FILE* f;
+
+    errno_t err = fopen_s(&f, "C:\\Users\\denes\\Git\\BiztonsagosGang\\parser\\caff_files\\2.CAFF", "r");
```

Comment:
Windows (Visual C compiler, to be precise) will complain, but `fopen_s` is not standard library function.

```suggestion
    FILE *f = fopen("C:\\Users\\denes\\Git\\BiztonsagosGang\\parser\\caff_files\\2.CAFF", "r");
``````

Reply:
The use of fopen causes compiling [error](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4996?f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(C4996)%26rd%3Dtrue&view=msvc-160)

Reply:
`#define _CRT_SECURE_NO_WARNINGS` in some core header or at the top of this file (from the same source)